source/Octopus.Client.Tests/Integration/HttpIntegrationTestBase.cs

```diff
@@ -57,12 +57,8 @@ public static void OneTimeSetup()
                 .UseContentRoot(Directory.GetCurrentDirectory())
                 .UseKestrel(o =>
                     {
-#if NET452
-                        o.UseHttps(GetCert());
-#else
```

Comment:
`net462` is ok without this?

Reply:
Yes,  by moving to .net 4.6.2 we have been able to consolidate our dependencies. Both our .Net Framework 4.6.2 and .Net standard tests make use of Kestrel 2.0 intead of use 1.1.2  for Framwork. That means this special case is not longer required (or works)